sdk/python/feast/repo_config.py

```diff
@@ -177,6 +177,14 @@ class RepoConfig(FeastBaseModel):
     go_feature_retrieval: Optional[bool] = False
     """ If True, use the embedded Go code to retrieve features instead of the Python SDK. """
 
+    go_transformations_server: Optional[bool] = True
```

Comment:
Had a look at the repo_config. Noticed feature_server section in repo_config. We should have grouped these configs there instead of separate configs.

Reply:
We are adding expedia provider. With that we can always force the flag to TRUE irrespective of what is set in feature_store.yaml. So we can ignore the above comment "Earlier logic was handling: Flag=False and Endpoint=Value, Its using Transformation service. Now it's using GOPY bindings"

Reply:
What about moving configurations to feature_server section?